holoviews/tests/plotting/plotly/test_barplot.py

```diff
@@ -91,3 +92,63 @@ def test_visible(self):
         element = Bars([3, 2, 1]).opts(visible=False)
         state = self._get_plot_state(element)
         self.assertEqual(state['data'][0]['visible'], False)
+    def test_bars_continuous_data_list_same_interval(self):
```

Comment:
```suggestion

    def test_bars_continuous_data_list_same_interval(self):
```

In the new test can you update to use assert, `np.testing`, etc. Instead of `self.assertEqual`